docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -515,6 +515,8 @@ spec:
   serviceUser: neutron
    ...
       customServiceConfig: |
+          [DEFAULT]
+          dhcp_agent_notification = True
```

Comment:
just a tiny, nitty nit: to be consistent you could use "true" (all lowercase) as in L521 or change L521 to be "True", same as here